cfgov/jinja2/v1/_includes/molecules/card.html

```diff
@@ -1,4 +1,17 @@
+{# ==========================================================================
+
+   Card
+
+   ==========================================================================
+
+   Description:
+
+   Creates a card with header and body text.
+
+   ========================================================================== #}
+
+{# TODO: This is a prototype and needs to change the text via parameters. #}
 <div class="m-card">
-  <h3 class="h1 m-card_heading">Card Heading</h3>
-  <p class="m-card_body">Card text</p>
+    <h3 class="m-card_heading h1">Card Heading</h3>
```

Comment:
Sure, I can revert it, I don't have a strong opinion on this. My thinking was that within the context of an atomic component the atomic class should always come first because everything else is essentially modifiers on that component.